db_migrate/versions/41c3d07202db_detection_status.py

```diff
@@ -0,0 +1,74 @@
+"""detection_status
```

Comment:
Migration has been turned off while v6 is in development. This file is not needed at this point.